### PR TITLE
Add terms of use and privacy policy

### LINKED ISSUE #439: Legal document adoption
We need a terms of service, privacy policy, etc.

Starting point: https://app.termly.io/

### PR DESCRIPTION
**Pull request by lmcnulty**
Resolves #439. Adds links in footer:

image

### CODE REVIEW

File: site/gatsby-site/content/privacy-policy.mdx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+### Links to Other Websites
+
+The Service may contain links to other websites not operated or controlled by the RAIC, including social media services (“Third Party Sites”). The information that you share with Third Party Sites will be governed by the specific privacy policies and terms of service of the Third Party Sites and not by this Privacy Policy. By providing these links we do not imply that we endorse or have reviewed these sites. Please contact the Third Party Sites directly for information on their privacy practices and policies.
+
+### Children
+
+Our Service is not directed to children who are under the age of 13. The RAIC does not knowingly collect Personal Information from children under the age of 13. If you have reason to believe that a child under the age of 13 has provided Personal Information to the RAIC through the Service please [contact us](https://incidentdatabase.ai/contact) and we will endeavor to delete that information from our databases.
```
Comment (smcgregor) [inline site/gatsby-site/content/privacy-policy.mdx:114]: Relative path.

File: site/gatsby-site/content/privacy-policy.mdx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+### Privacy Policy Changes
+
+The Service and/or our activities and businesses may change from time to time. Please review this policy periodically. As a result we may change this Privacy Policy at any time. When we do we will post an updated version on this page, unless another type of notice is required by the applicable law. By continuing to use our Service or providing us with Personal Information after we have posted an updated Privacy Policy, or notified you by other means if applicable, you consent to the revised Privacy Policy and practices described in it.
+
+### How to Reach Us
+
+If you have a question about this Privacy Policy, or you would like to contact us about any of the rights mentioned in the above, please [contact us](https://incidentdatabase.ai/contact).
```
Comment (smcgregor) [inline site/gatsby-site/content/privacy-policy.mdx:126]: Relative path

File: site/gatsby-site/content/terms-of-use.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+### Privacy Policy
+At RAIC, we respect the privacy of our users. For details please see our [Privacy Policy](/privacy-policy). By using the Site, you consent to our collection and use of personal data as outlined therein.
+
+If you have any privacy-related questions or unresolved problems, you may [contact us](https://incidentdatabase.ai/contact).
```
Comment (smcgregor) [inline site/gatsby-site/content/terms-of-use.mdx:93]: Relative path

File: site/gatsby-site/src/templates/docs.js [hunk 4]
```diff
@@ -7,8 +7,10 @@ import Layout from 'components/Layout';
 import { StyledHeading, StyledMainWrapper } from 'components/styles/Docs';
 import config from '../../config';
 import { MDXProvider } from '@mdx-js/react';
+import isString from 'lodash/isString';
 
-const slug = (title) => title.toLowerCase().replace(/\s+/g, '');
+const slug = (title) =>
```
[the remark this answers is not in the document]
Comment (smcgregor) [reply site/gatsby-site/src/templates/docs.js:12]: I believe titles are also used as page titles, which I don't believe supports markup. I don't see markup in the titles of this PR...

[the remark this answers is not in the document]
Comment (lmcnulty) [reply site/gatsby-site/src/templates/docs.js:12]: This function is used to get heading slugs. Some of the headings on the added pages are italicized in the markdown. E.g. `#### _European General Data Protection Regulation (GDPR)_`.

[the remark this answers is not in the document]
Comment (smcgregor) [reply site/gatsby-site/src/templates/docs.js:12]: ahhh, got it, thanks

**Review by smcgregor — APPROVED** [review]

**Pull request merged**